src/test/game/CommandLineUITest.java

```diff
@@ -17,17 +18,35 @@
     private ByteArrayOutputStream output;
     private  InputStream input;
     private CommandLineUI ui;
+    private English english;
 
     @Before
     public void setUp() {
         output = new ByteArrayOutputStream();
         input = new ByteArrayInputStream("".getBytes());
         ui = new CommandLineUI(new PrintStream(output), input);
+        english = new English();
+    }
+
+    @Test
+    public void askUserForLanguageSelection() {
+        ui.askUserForLanguageSelection();
+
+        assertTrue(output.toString().contains("Select your language:\nEnter '1' for English\nΕισαγάγετε '2' για τα Ελληνικά"));
+    }
+
+    @Test
+    public void getLanguageSelection() {
+       InputStream input = new ByteArrayInputStream("1".getBytes());
+
+       CommandLineUI ui = new CommandLineUI(new PrintStream(output), input);
+
+        assertEquals("1", ui.getLanguageSelection());
```

Comment:
Make sure you leverage your IDE's ability to reformat your code to avoid stray spaces like this one.